lib/private/App/AppManager.php

```diff
@@ -148,6 +149,27 @@ public function getEnabledAppsForUser(IUser $user) {
 		return array_keys($appsForUser);
 	}
 
+	/**
+	 * @param \OCP\IGroup $group
+	 * @return array
+	 */
+	public function getEnabledAppsForGroups($group) {
+		$apps = $this->getInstalledAppsValues();
+		$appsForGroups = array_filter($apps, function ($enabled) use ($group) {
+			return $this->checkAppForGroups($enabled, $group);
+		});
+		return array_keys($appsForGroups);
+	}
+	public function getAppRestriction($appname) {
```

Comment:
```suggestion
	public function getAppRestriction(string $appId): array {
```

Also use `$appId` inside please, so it's consistent with the remaining code